guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -200,38 +200,52 @@
-        "ERROR_EMPTY_FILE"               : "The provided file is empty",
-        "ERROR_INVALID_CSV_HEADER"       : "Invalid CSV Header \"{HEADER}\" is neither an attribute or parameter",
-        "ERROR_INVALID_MIME_TYPE"        : "Unsupported file type: \"{TYPE}\"",
-        "ERROR_DETECTED_INVALID_TYPE"    : "Unsupported file type. Please make sure the file is valid CSV, JSON, or YAML.",
-        "ERROR_INVALID_GROUP"            : "No group matching \"{GROUP}\" found",
-        "ERROR_INVALID_GROUP_IDENTIFIER" : "No connection group with identifier \"{IDENTIFIER}\" found",
-        "ERROR_NO_FILE_SUPPLIED"         : "Please select a file to import",
-        "ERROR_PARSE_FAILURE_CSV"        : "Please make sure your file is valid CSV. Parsing failed with error \"{ERROR}\". ",
-        "ERROR_PARSE_FAILURE_JSON"       : "Please make sure your file is valid JSON. Parsing failed with error \"{ERROR}\". ",
-        "ERROR_PARSE_FAILURE_YAML"       : "Please make sure your file is valid YAML. Parsing failed with error \"{ERROR}\". ",
-        "ERROR_REQUIRED_NAME"            : "No connection name found in the provided file",
-        "ERROR_REQUIRED_PROTOCOL"        : "No connection protocol found in the provided file",
+        "ERROR_AMBIGUOUS_CSV_HEADER"         : "Ambiguous CSV Header \"{HEADER}\" could be either a connection attribute or parameter",
+        "ERROR_AMBIGUOUS_PARENT_GROUP"       : "Both group and parentIdentifier may be not specified at the same time",
+        "ERROR_ARRAY_REQUIRED"               : "The provided file must contain a list of connections",
+        "ERROR_DETECTED_INVALID_TYPE"        : "Unsupported file type. Please make sure the file is valid CSV, JSON, or YAML.",
+        "ERROR_DUPLICATE_CONNECTION_IN_FILE" : "Duplicate connection in file at \"{PATH}\"",
+        "ERROR_DUPLICATE_CSV_HEADER"         : "Duplicate CSV Header: {HEADER}",
+        "ERROR_EMPTY_FILE"                   : "The provided file is empty",
+        "ERROR_INVALID_CSV_HEADER"           : "Invalid CSV Header \"{HEADER}\" is neither an attribute or parameter",
+        "ERROR_INVALID_MIME_TYPE"            : "Unsupported file type: \"{TYPE}\"",
+        "ERROR_INVALID_GROUP"                : "No group matching \"{GROUP}\" found",
+        "ERROR_INVALID_GROUP_IDENTIFIER"     : "No connection group with identifier \"{IDENTIFIER}\" found",
+        "ERROR_NO_FILE_SUPPLIED"             : "Please select a file to import",
+        "ERROR_PARSE_FAILURE_CSV"            : "Please make sure your file is valid CSV. Parsing failed with error \"{ERROR}\". ",
+        "ERROR_PARSE_FAILURE_JSON"           : "Please make sure your file is valid JSON. Parsing failed with error \"{ERROR}\". ",
+        "ERROR_PARSE_FAILURE_YAML"           : "Please make sure your file is valid YAML. Parsing failed with error \"{ERROR}\". ",
+        "ERROR_REJECT_UPDATE_CONNECTION"     : "Disallowed update to existing connection at \"{PATH}\"",
```

Comment:
As written, this makes sense to me only in the context of an update, but:

* A connection import isn't an update unless the "Replace/Update ..." box is checked.
* This error doesn't occur if the "Replace/Update" box is checked.

Assuming the name can be included, recommend rephrasing to: `"Connection \"{NAME}\" already exists at \"{PATH}\"."`